src/devtools/Explorer.tsx

```diff
@@ -45,43 +52,81 @@ export const Expander = ({ expanded, style = {}, ...rest }) => (
   </span>
 )
 
-const DefaultRenderer = ({
-  handleEntry,
+type Entry = {
+  label: string
+}
+
+type RendererProps = {
+  HandleEntry: HandleEntryComponent
+  label?: string
+  value?: any
```

Comment:
can we use `unknown` here instead of `any`?

Reply:
Added, I needed to add an extra check that the value isIterable

5bc51d8949ec0be5f6e6cf9811f344ec198ff6f6